<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    #ROS#
    <param name="scan_topic" value="scan"/>
    <param name="cloud_topic" value="cloud"/>
    <param name="frame_id" value="map" />
     #DATA#
    <param name="min_dist" value="0.01"/>
    <param name="max_dist" value="50.0"/>
    <param name="output_scan" value="true" />
    <param name="inverted" value="true" />
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="180.0"/>
    <param name="max_angle" value="225.0"/> 
    #CONNECT#
    <param name="type" value="uart" />
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud_rate" value="230400" />

  </node>
 </group>
</launch>

